### PR TITLE
fix(ui): Remove -webkit focus outline on radio inputs

### PR DESCRIPTION
**Pull request by vuluongj20**
Before:
Screen Shot 2022-01-12 at 11 29 03 AM

After:
Screen Shot 2022-01-12 at 11 29 26 AM

### CODE REVIEW

File: static/app/components/radio.tsx [hunk 1]
```diff
@@ -38,8 +38,8 @@ const Radio = styled('input')<Props>`
   /* TODO(bootstrap): Our bootstrap CSS adds this, we can remove when we remove that */
   margin: 0 !important;
 
-  &:focus,
-  &.focus-visible {
+  &&:focus,
+  &&.focus-visible {
```
Comment (evanpurkhiser) [inline static/app/components/radio.tsx:42]: 🤔  wait how does this work

File: static/app/components/radio.tsx [hunk 2]
```diff
@@ -38,8 +38,8 @@ const Radio = styled('input')<Props>`
...
     outline: none;
```
Comment (vuluongj20) [inline static/app/components/radio.tsx:43]: The current selector isn't specific enough to override the default outline.

File: static/app/components/radio.tsx (hunk 1 above)
Comment (vuluongj20) [reply static/app/components/radio.tsx:42]: @evanpurkhiser ⬇️

File: static/app/components/radio.tsx (hunk 2 above)
Comment (vuluongj20) [reply static/app/components/radio.tsx:43]: The focus outline comes from `mixing/tab-focus.less`. Ideally we should just remove this file, but I'm not sure if there are templates that rely on it.

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**